internal/service/streamconnection/resource_stream_connection_test.go

```diff
@@ -235,6 +235,62 @@ func TestAccStreamPrivatelinkEndpoint_streamConnection(t *testing.T) {
 	})
 }
 
+func TestAccStreamRSStreamConnection_AWSLambda(t *testing.T) {
+	var (
+		resourceName   = "mongodbatlas_stream_connection.test"
+		projectID      = os.Getenv("MONGODB_ATLAS_ASP_PROJECT_EAR_PE_ID") //test-acc-tf-p-keep-ear-AWS-private-endpoint project has aws integration
```

Comment:
@oarbusi can we add this env var? `test-acc-tf-p-keep-ear-AWS-private-endpoint` project has aws integration with `arn:aws:iam::358363220050:role/mongodb-atlas-test-acc-tf-7973337217371171538-git-ear` role. So I guess we can use that project. The value of below env var `MONGODB_ATLAS_ASP_PROJECT_AWS_ROLE_ARN` would be `arn:aws:iam::358363220050:role/mongodb-atlas-test-acc-tf-7973337217371171538-git-ear`.

Reply:
@oarbusi the project ID is `6790e57a9b41416f5c216fee`. Can you add it? Thanks.